aerobites/src/AirportDetails/AirportDetail.js

```diff
@@ -1,19 +1,46 @@
-import React from 'react';
+import { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import { useParams, Link } from 'react-router-dom';
+import { getTerminals, getBusinesses } from '../ApiCalls/ApiCalls';
 
 export default function AirportDetails({ airports, toggleFavorite }) {
     // get airport name from the URL using useParams
-    const { airportName } = useParams();
-    const decodedName = decodeURIComponent(airportName);
-    const airport = airports.find(a => a.name === decodedName);
+    const preairportId = useParams().Id
+    const airportId = parseInt(preairportId)
```

Comment:
`preairportId` is only used on line 9, you could consider doing something like `parseInt(useParams().Id)` to consolidate some code and save a variable assignment